### ISSUE TITLE
favicon.ico 404

### ISSUE THREAD

**Issue by nathanweeks** [legumeinfo/jekyll-starter-legumeinfo#9]
The theme expects favicon.ico to be at /assets/favicon.ico:

```
docker compose up
... point browser to http://localhost:4000 ...
ERROR `/assets/favicon.ico' not found.
```

This starter site has the favicon at /assets/icons/favicon.ico

**Comment by alancleary** [comment]
Mind opening a PR for this one?

In general the theme should be checking if the various icon files existing and only adding link tags for them if they do. I'll open an issue in the theme repo for that. **Edit:** See [issue #52](https://github.com/legumeinfo/jekyll-theme-legumeinfo/issues/52).

**Comment by alancleary** [comment]
This was addressed in [theme PR #53](https://github.com/legumeinfo/jekyll-theme-legumeinfo/pull/53).
